### PR TITLE
Fixes the isSelected function to work if the value is equal

### PR DESCRIPTION
**Pull request by Rdbaker**
## Problem

I had noticed that, when using `react-big-calendar` I would pass the event that was selected, but the rendered component would not be selected (the `selected` prop was `false`).

Upon investigating, I found it to be an issue with the `isSelected` function. I wrote tests to confirm this.

Here is the output of the tests before the code change:

```
√:react-big-calendar rdbaker $ yarn test
yarn run v1.22.5
$ yarn lint && jest
$ eslint src test
Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
 PASS  test/utils/TimeSlots.test.js
 PASS  test/utils/eventLevels.test.js (5.138s)
 PASS  test/utils/DayEventLayout.test.js (5.265s)
 FAIL  test/utils/selection.test.js
  ● isSelected › it returns true if it is the same object by equality

    expect(received).toBeTruthy()

    Received: false

      23 |       y: 1
      24 |     }
    > 25 |     expect(isSelected(value, equalivalentValue)).toBeTruthy()
         |                                                  ^
      26 |   })
      27 |
      28 |   test('it returns false if the object is not equal', () => {

      at Object.<anonymous> (test/utils/selection.test.js:25:50)

Test Suites: 1 failed, 3 passed, 4 total
Tests:       1 failed, 75 passed, 76 total
Snapshots:   0 total
Time:        7.397s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

And here is the output after the code change:

```
?1:react-big-calendar rdbaker $ yarn test
yarn run v1.22.5
$ yarn lint && jest
$ eslint src test
Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
 PASS  test/utils/DayEventLayout.test.js
 PASS  test/utils/eventLevels.test.js
 PASS  test/utils/TimeSlots.test.js
 PASS  test/utils/selection.test.js

Test Suites: 4 passed, 4 total
Tests:       76 passed, 76 total
Snapshots:   0 total
Time:        3.206s, estimated 6s
Ran all test suites.
✨  Done in 10.27s.
```

I didn't find a doc on how to structure contributions to this repo, so I laid it out as clearly as I could (granted, I didn't look at past merged contributions, I can take a look at that). Feel free to mark this up with comments/edits.

### CODE REVIEW

File: src/utils/selection.js [hunk 1]
```diff
@@ -18,7 +20,7 @@ export function getSlotAtX(rowBox, x, rtl, slots) {
 }
 
 export function pointInBox(box, { x, y }) {
-  return y >= box.top && y <= box.bottom && (x >= box.left && x <= box.right)
+  return y >= box.top && y <= box.bottom && x >= box.left && x <= box.right
```
Comment (Rdbaker) [inline src/utils/selection.js:23]: This was done automatically, I assume by the linting config that's set up. I'm happy to revert this change if that's preferable.

**Comment by Rdbaker** [comment]
@pietrofxq any chance you have time for a quick 👀 on this? It's ~2 lines of functional changes and ~40 lines of 3 tests

**Comment by pietrofxq** [comment]
Looks good, @Rdbaker can you just change the commit message to follow convetional commit (just add fix: )

**Review by pietrofxq — APPROVED** [review]

**Comment by Rdbaker** [comment]
@pietrofxq done!
